test/Tests/Attributes/MetaData/IgnoreAttributeTests.cs

```diff
@@ -22,12 +37,19 @@ public void Enum_Members_Can_Be_Ignored()
             type.ShouldHaveFieldWithName("DEPRECATED_MEMBER").AndWithDeprecationReason("Some enum member reason");
         }
 
+        public void Dispose()
```

Comment:
See note above about not needing `Reset*` methods if the functionality is exposed through `ITypeResolver`